lisa/util/__init__.py

```diff
@@ -231,3 +232,20 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
 
 def filter_ansi_escape(content: str) -> str:
     return __ansi_escape.sub("", content)
+
+
+def dump_files(os, environment) -> List[TextIOWrapper]:
+    if len(str(environment.log_path).split("\\environments\\")) < 2:
```

Comment:
1. The `/` and `\` can be handled correctly by pathlib.Path. Search pathlib in code to find some examples.
2. As above comments, make this method generic. In this method, it should check if the files are in `constant.RUN_LOCAL_PATH`.